lib/questionnaires/model/item/answer/src/coding_answer_option_model.dart

```diff
@@ -132,7 +133,11 @@ class CodingAnswerOptionModel {
         ?.valueString;
 
     final optionPrefix = (plainOptionPrefix != null)
-        ? RenderingString.fromText(plainOptionPrefix)
+        ? RenderingString.fromText(
+            plainOptionPrefix,
+            locale: locale,
+            extensions: extensions,
```

Comment:
Just leaving this comment here so that we don't forget to change this.

Reply:
Thanks for the comment, addressed [here](https://github.com/sujrd/faiadashu/pull/49/commits/3598028e169d969b943d32e8b739ed21aebcca11).